src/sundials/sundials_math.c

```diff
@@ -36,8 +36,6 @@ sunrealtype SUNRpowerI(sunrealtype base, int exponent)
 
 sunrealtype SUNRpowerR(sunrealtype base, sunrealtype exponent)
 {
-  if (base <= SUN_RCONST(0.0)) { return (SUN_RCONST(0.0)); }
```

Comment:
I went through every reference to this function across SUNDIALS and did not see a case where `base` could be negative. There are a few cases where `base` could be 0, but `exponent` is finite so the behavior is unchanged.